mountpoint-s3/tests/common/s3.rs

```diff
@@ -54,6 +54,10 @@ pub async fn get_test_sdk_client(region: &str) -> aws_sdk_s3::Client {
     aws_sdk_s3::Client::new(&sdk_config)
 }
 
+pub fn get_test_kms_key_arn() -> String {
+    std::env::var("KMS_TEST_KEY_ARN").expect("Set KMS_TEST_KEY_ARN to run integration tests")
+}
```

Comment:
we can follow up later, but I'd prefer we take just ARN and transform it (drop ARN prefix) where we need an ID.

Reply:
I'll abandon this PR for now as an alternative approach was decided to be taken, we can revisit this in feature